offline/cable_cru_TRENDY.F90

```diff
@@ -451,18 +636,43 @@ SUBROUTINE cru_get_daily_met(CRU, LastDayOfYear)
     DummyYear = CRU%cYear + 1
   END IF
 
-  ! Was the Tmin recycled?
+  ! Now we have to handle each of the "next" variables individually
+  ! Tmax
   IF (CRU%isRecycled(Tmin)) THEN
```

Comment:
Why does it say Tmax in the comment but looks at "isRecycled(**Tmin**)"?

Reply:
Looks like I got a little mixed up here- need to get the ```vph1500``` from the previous day and ```vph0900``` from the next. I was retrieving the right data, but checking against the wrong recycling logical.